Program/stud1.txt

```diff
@@ -0,0 +1,41 @@
+[1] Create table stud(id, name, dept, paidfee) with 8 records. Create stud.csv file of the stud data (Department will be BBA and BCA). 
+   
+   Read csv file using reader object and write data with remaining fee. ( Total fee of bba is 13400 and bca is 15400 Rs)
+
+
+sqlite> .open student
+
+sqlite> create table stud1
+   ...> (sid int primary key,sname text,dept,paid_fee);

```

Comment:
Is it necssary to add datatype here ?

Reply:
> Is it necssary to add datatype here ?

No its not necessary in sqlite3. But we can add datatype for better understanding of us.